src/CallHandler.js

```diff
@@ -301,23 +301,23 @@ function _onAction(payload) {
         case 'place_conference_call':
             console.log("Place conference call in %s", payload.room_id);
 
-            if (MatrixClientPeg.get().isRoomEncrypted(payload.room_id)) {
-                // Conference calls are implemented by sending the media to central
-                // server which combines the audio from all the participants together
-                // into a single stream. This is incompatible with end-to-end encryption
-                // because a central server would be decrypting the audio for each
-                // participant.
-                // Therefore we disable conference calling in E2E rooms.
-                const ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
-                Modal.createTrackedDialog('Call Handler', 'Conference calls unsupported e2e', ErrorDialog, {
-                    description: _t('Conference calls are not supported in encrypted rooms'),
-                });
-                return;
-            }
-
             if (SettingsStore.isFeatureEnabled('feature_jitsi')) {
```

Comment:
if jitsi is not enabled, the reason we don't start the call is not really that it's not supported in e2e encrypted rooms, is it? Should we change the error message? Or is this about freeswitch calls not working in e2e encrypted rooms?

Reply:
yep, if jitsi's not enabled then we use freeswitch which we haven't supported in e2e rooms since it would just send your media to freeswitch which isn't very e2e. (Jitsi is no different but there's a separate general warning for widgets)